mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -2085,7 +2091,7 @@ impl TakerSwap {
         taker_coin: MmCoinEnum,
         swap_uuid: &Uuid,
     ) -> Result<(Self, Option<TakerSwapCommand>), String> {
-        let saved = match SavedSwap::load_my_swap_from_db(&ctx, *swap_uuid).await {
+        let saved = match SavedSwap::load_my_swap_from_db(&ctx, None, *swap_uuid).await {
```

Comment:
Why None here? this is part of the kickstart process if I am not mistaken. At least leave a todo.

Reply:
??
None is just a marker to fetch the address_dir from the global DB.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2398#discussion_r2090485875